mdbenchmark/mdengines/gromacs.py

```diff
@@ -19,11 +19,13 @@
 # along with MDBenchmark.  If not, see <http://www.gnu.org/licenses/>.
 import os
 from shutil import copyfile
+import string
 
 from mdbenchmark import console
 
 NAME = "gromacs"
 
+MULTIDIRS=string.ascii_lowercase
```

Comment:
Let's be more specific here and use "LOWERCASE_LETTERS" to make the usage more explicit.

```suggestion
LOWERCASE_LETTERS = string.ascii_lowercase
```